.travis.yml

```diff
@@ -15,7 +15,7 @@ before_install:
     openssl aes-256-cbc -K $encrypted_227846deebd3_key -iv $encrypted_227846deebd3_iv -in .travis/secrets.tar.enc -out .travis/local.secrets.tar -d; tar xv -C .travis -f .travis/local.secrets.tar;
   fi
 script:
-- sbt ++$TRAVIS_SCALA_VERSION clean coverage test scalafmtCheck scalafmtSbtCheck
+- sbt ++$TRAVIS_SCALA_VERSION clean coverage +test scalafmtCheck scalafmtSbtCheck
```

Comment:
to build multiple versions you could also list them above in `scala`. Something like https://github.com/laserdisc-io/laserdisc/blob/master/.travis.yml#L5